pyartcd/pyartcd/runtime.py

```diff
@@ -3,7 +3,7 @@
 from pathlib import Path
 from typing import Any, Dict, Optional
 
-import toml
+import tomli
```

Comment:
This change is not related but let me explain why it is here.

When I attempted to store UMB brokers configuration in artcd.toml, I met some bugs in
`toml` library. It can't correctly parse configs with nested dicts and arrays.

Considering `toml` is not actively maintained and `tomli` has been adopted as a standard library in Python 3.11, I think we should move to `tomli`.

At the end, I decided to write a URI parser to configure UMB brokers using syntax like `stomp+ssl://example.com:12345`, so this became an irrelevant change.